src/components/icons/user-activities/item-edited.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Seems like this is the only one that does not have a file name ending in `-icon.js`

Reply:
Good catch, thanks! Corrected